src/backend/aspen/api/views/phylo_trees.py

```diff
@@ -0,0 +1,51 @@
+import sentry_sdk
+import sqlalchemy as sa
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.orm import aliased, joinedload
+from sqlalchemy.sql.selectable import Select
+
+from aspen.api.auth import get_auth_user
+from aspen.api.deps import get_db
+from aspen.api.error import http_exceptions as ex
+from aspen.api.schemas.phylo_trees import PhyloTreeRequest, PhyloTreeResponse
+from aspen.database.models import Group, PhyloRun, PhyloTree, User
+
+router = APIRouter()
+
+
+@router.put("/")
```

Comment:
I think the tree ID needs to be part of the route here, rather than part of the request body.

Reply:
makes sense, i'll change it